### PR TITLE
DEX-246 Search API for individuals

### PR DESCRIPTION
**Pull request by mmulich**
## Pull Request Overview

- Adds elasticsearch and logstash to the composition
- Adds [gumby](https://github.com/wildmeorg/gumby) library to the project
- Provides configuration for loading the `individuals` index from the EDM database using logstash
- Provides an endpoint for proxying elasticsearch requests through houston
- Adjusts the integrations checks to include elasticsearch

---

**Review Notes**

It's difficult to see the results of this work thus far. If everything is running properly and edm has some data in it, then maybe you'll see results in a search. For example, you could run `echo '{"query": {"match_all": {}}}' | http -v POST localhost:9200/individuals/_search`. (Though, in most cases this isn't going to give back anything because the edm has no data in it.)

If you'd artificially like to load the EDM with data (get the `giraffe-ng.sql` from Jon). The do the following:
<details>
<summary>instructions for loading edm with data</summary>
```
    alias dc=docker-compose
    # drop the existing schema and data
    dc exec db psql -U wildbook -d wildbook -c "drop schema public cascade; create schema public;"
    # load the database
    cat ng-giraffe.sql | dc exec -T db psql -U wildbook -d wildbook
    # verify the import...
    dc exec db psql -U wildbook -d wildbook -c "select count(*) from \"MARKEDINDIVIDUAL\";"
    # count should be 6437
```
</details>

**REST API Updates**

This primarily adds the `/api/v1/search/<string:index>` endpoint. The goal at this point is to simply proxy the request from to elasticsearch. Later adjustments will likely lock this down.

For example, the following is a search against the `individuals` index that searches for all records:
```
POST /api/v1/search/individuals
{"query": {"match_all": {}}}
```

**Invoke CLI Updates**

The `app.integrations.check` has been updated to include checking the connection to elasticsearch.

### CODE REVIEW

**Comment by mmulich** [comment]
Last check all things are passing/green. So I've opened this up for review.

File: app/requirements.txt [hunk 1]
```diff
@@ -26,6 +26,7 @@ flask-restx>=0.2.0
 Flask-SQLAlchemy>=2.4,<3
 
 gitpython
+gumby @ git+https://github.com/WildMeOrg/gumby.git
```
Comment (colinwkingen) [inline app/requirements.txt:29]: Does this need to be added to existing local repos like a submodule, or will a docker-compose.rebuild bring it in?

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -82,7 +82,7 @@ jobs:
         run: |
           set -ex
           cd deploy/codex
-          docker-compose up -d db redis gitlab edm houston acm
+          docker-compose up -d db redis gitlab edm houston acm elasticsearch
```
Comment (colinwkingen) [inline .github/workflows/testing.yml:85]: Do you have a ballpark on resource consumption? Probably not much... until we get a big ol' database I imagine.

**Review by colinwkingen — APPROVED** [review]
A couple small questions, but this looks good to me. Thanks!

**Pull request merged**